### PR TITLE
5834 - Convert area protractor test suites to puppeteer

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR converts the test suites from protractor to puppeteer.

**Related github/jira issue (required)**:

Closes https://github.com/infor-design/enterprise/issues/5834

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Run `npm run e2e:puppeteer area`
- All tests should pass
- Verify and check the code implementation to puppeteer

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Comment by ericangeles** [comment]
Expecting the build to fail. The fix is [here](https://github.com/infor-design/enterprise/pull/5993).

File: test/components/area/area.puppeteer-spec.js [hunk 1]
```diff
@@ -1,8 +1,102 @@
...
+      const emptyTitle = await page.waitForSelector('.empty-title', { visible: true });
+
+      expect(emptyMsg).toBeTruthy();
+      expect(await emptyTitle.evaluate(el => el.textContent)).toContain('No Data Available');
+    });
+  });
+
   describe('Area Disable Selection  State Tests', () => {
-    const url = 'http://localhost:4000/components/area/example-disable-selection-state.html';
+    const url = `${baseUrl}/example-disable-selection-state`;
```
Comment (tmcconechy) [inline test/components/area/area.puppeteer-spec.js:99]: Nice tests! Just seems like this test doesnt have any checks?

Comment (ericangeles) [reply test/components/area/area.puppeteer-spec.js:99]: What do you mean about checks @tmcconechy ?

Comment (tmcconechy) [reply test/components/area/area.puppeteer-spec.js:99]: my mistake @ericangeles you can ignore that comment. When i looked at the diff i didnt see an `it(` statements but i see they are there in the file just not the diff so i thought it had no checks/tests for a minute

**Review by tmcconechy — APPROVED** [review]

**Review by EdwardCoyle — APPROVED** [review]

**Pull request merged**